reference/constraints/Traverse.rst

```diff
@@ -84,15 +116,23 @@ configure the ``Traverse`` constraint on the ``Book`` class.
         use Symfony\Component\Validator\Constraints as Assert;
         use Symfony\Component\Validator\Mapping\ClassMetadata;
 
-        class Book
+        class BookCollection
```

Comment:
I can't comment on collapsed lines (https://github.com/isaacs/github/issues/1655), I wanted to suggest on old line 81:
```diff
-        // src/AppBundle/Entity/Book.php
+        // src/AppBundle/Entity/BookCollection.php
```

Reply:
Good catch! Thanks